ProcessLib/StaggeredCouplingTerm.h

```diff
@@ -0,0 +1,42 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   StaggeredCouplingTerm.h
+ *
+ * Created on November 7, 2016, 12:14 PM
+ */
+
+#ifndef OGS_STAGGERED_COUPLING_TERM_H
+#define OGS_STAGGERED_COUPLING_TERM_H
+
+#include <map>
+
+#include "MathLib/LinAlg/GlobalMatrixVectorTypes.h"
+
+#include "ProcessType.h"
+
+namespace ProcessLib
+{
+class Process;
+struct StaggeredCouplingTerm
+{
+    StaggeredCouplingTerm(
+        std::map<ProcessType, Process const&> const& coupled_processes_,
```

Comment:
Already replaced ``std::map`` with ``std::unordered_map``, and ``ProcessType`` with ``std::type_index`` in another commit.